modules/checkout/commerce_checkout.module

```diff
@@ -34,6 +34,13 @@ function commerce_checkout_theme() {
     'commerce_checkout_form__with_sidebar' => [
       'base hook' => 'commerce_checkout_form',
     ],
+    'commerce_checkout_order_summary' => [
+      'variables' => [
+        'order' => NULL,
```

Comment:
Other commerce_checkout and commerce_order templates use order_entity instead of order.